constraint/pkg/client/client.go

```diff
@@ -767,6 +785,25 @@ func (c *Client) Dump(ctx context.Context) (string, error) {
 	return dumpBuilder.String(), nil
 }
 
+func (c *Client) GetDescriptionForStat(source instrumentation.Source, statName string) string {
+	if source.Type != instrumentation.EngineSourceType {
+		// only handle engine source for now
+		return instrumentation.UnknownDescription
+	}
+
+	driver, ok := c.drivers[source.Value]
+	if !ok {
+		return instrumentation.UnknownDescription
+	}
+
+	desc, err := driver.GetDescriptionForStat(statName)
+	if err != nil {
```

Comment:
do we want to bubble this error up?

Reply:
I could be persuaded to bubble the error up. I know that Max made a comment to bubble up the error from the `driver.GetDescriptionForStat`. 

For the client UX, I thought it was better for the emission (logging) to not have to handle the error and always get in hand a string, the description requested or an "unknown description" string.

Reply:
IMO we should go all-in one direction or the other (either bubble up the error all the way or don't raise an error). 

For now, I'm okay with leaving this as-is, we can revamp as we learn more about how this is used.